demo/src/components/blog.js

```diff
@@ -0,0 +1,39 @@
+/* global window */
+import React, {Component, PropTypes} from 'react';
+import autobind from 'autobind-decorator';
+
```

Comment:
I think it is worth declaring the propTypes. It really helps when reading the code.